source/documentation/other-topics/custom-domain-cert.md

```diff
@@ -48,18 +72,40 @@ material will be stored, in your namespace.
        - domains:
          - '<hostname>'
          dns01:
-           provider: <provider>
+           provider: route53-cloud-platform
    ```
 
-3. Make sure the certificate has been issued correctly, by checking its `Status`:
+2. Make sure the certificate has been issued correctly, by checking its `Status`:
 
    ```
    $ kubectl describe certificate <my-cert>
    ```
 
-4. You will need to update your `Ingress` spec to include the new hostname.
+   The certificate status of type `"Ready"` should be `True`:
+
+   ```
+   Status:
+     Conditions:
+       Last Transition Time:  2019-06-05T10:16:43Z
+       Message:               Certificate is up to date and has not expired
+       Reason:                Ready
+       Status:                True
+       Type:                  Ready
+     Not After:               2019-09-03T09:16:42Z
+   Events:
+     Type    Reason         Age   From          Message
+     ----    ------         ----  ----          -------
+     Normal  Generated      3m    cert-manager  Generated new private key
+     Normal  OrderCreated   3m    cert-manager  Created Order resource "<my-cert>-3189350212"
+     Normal  OrderComplete  1m    cert-manager  Order "<my-cert>-3189350212" completed successfully
+     Normal  CertIssued     1m    cert-manager  Certificate issued successfully
+   ```
+
+   It generally takes but a few minutes for the certificate to be prepared and
+   the events displayed should indicate if there is a problem or it simply needs
+   more time.
 
-   **Once your host is defined here, the cluster will take control of the DNS record and automatically adjust to point to the cluster.**
+3. You will need to update your `Ingress` spec to include the new hostname.
```

Comment:
I think this line "3. You will need to update your ingress..." should be moved below the following paragraph - ie. so that it is directly before the relevant code sample.

Reply:
Oops. Thanks!

In fact, the note on manually intervention was relevant only for existing zones we could be re-using. I've removed it and hopefully any confusion with it.